definitions/private-law/json/FixedLists.json

```diff
@@ -4324,29 +4324,29 @@
   {
     "LiveFrom": "01/12/2022",
     "ID": "DioBeforeAEnum",
-    "ListElementCode": "legalAdviser",
-    "ListElement": "Legal adviser",
+    "ListElementCode": "circuitJudge",
+    "ListElement": "Circuit judge",
```

Comment:
why this change? any change in this file?

Reply:
this changes has been reverted